docker/php/config/parameters.yml

```diff
@@ -6,7 +6,7 @@ parameters:
     database_name: ${DATABASE_NAME:-symfony}
     database_user: ${DATABASE_USER:-root}
     database_password: ${DATABASE_PASSWORD:-~}
-    database_path: '${DATABASE_PATH:-"%kernel.root_dir%/data/db/wallabag.sqlite"}'
+    database_path: '${DATABASE_PATH:-"%kernel.project_dir%/data/db/wallabag.sqlite"}'
```

Comment:
looks like the previous default value was broken as root_dir is the directory where the kernel is, so `app/`, while `data/` is at the root of the project, not inside `app/`